runtime/gc_vlhgc/AllocationContextBalanced.hpp

```diff
@@ -72,6 +72,7 @@ class MM_AllocationContextBalanced : public MM_AllocationContextTarok
 	UDATA _freeProcessorNodeCount;	/**< The length, in elements, of the _freeProcessorNodes array (always at least 1 after startup) */
 
 #if defined(J9VM_GC_SPARSE_HEAP_ALLOCATION)
+	intptr_t _fractionReservedRegion;
 	MM_HeapRegionDescriptorVLHGC *_arrayReservedRegionList; /** for Off-heap case only, adding and removing via region->_allocateData.pushRegionToArrayReservedRegionList/popRegionFromArrayReservedRegionList */
```

Comment:
change the comment to: the list of all whole regions reserved in main heap to offset the allocation in the off-heap